starters/next-drupal-starter/pages/index.js

```diff
@@ -28,9 +28,9 @@ export default function Home({ articles, hrefLang, multiLanguage }) {
 
           <div className="text-2xl">
             <div className="bg-black text-white rounded flex items-center justify-center p-4">
-              Decoupled hosting by{" "}
+              Decoupled Drupal on{" "}
               <Image
-                src="/pantheon.svg"
+                src="/pantheon.png"
```

Comment:
The fallback image below (line 72) and on the /articles page also needs the .png switch

Reply:
Whoops, I think I got the rest of them.